docs/README.md

```diff
@@ -83,41 +83,47 @@ The Notebooks can run on various compute targets; the ones currently supported o
 
 # Productionalization
 
-Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. One of the main reasons being that feature engineering (featurization) requires not only new, but also historic data. For reasons outlined below, this solution was built to support real-time featurization and scoring.
+Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. As it is often the case when dealing with time series, feature engineering (featurization) requires access to historical data in addition to the new data.
```

Comment:
I'm not a fan of this phrasing.  We're basically saying "this is a hard problem" to an audience of people that are seeking guidance to solve the problem.  I would recommend providing more specific and actionable items in this section, even if those items are redirection to other documentation.

While designing your production data pipeline for a Predictive Maintenance solution, be aware that Feature engineering (featurization) requires access to historical data as well as real-time data streams.